game/theater/start_generator.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+import qt_ui.uiconstants as CONST
```

Comment:
Never import the UI from the backend. If you need something from the UI in `game` or `gen` the constant needs to be moved.

This also appears unused?

Reply:
Remnant of the old "None, 25%..." box. Removed.